### PR TITLE
Setup config files for flake8

### LINKED ISSUE #9: Setup .travis.yml to run flake8
We should add flake8 to test-requirements.txt and write a .travis.yml to run flake8.
The flake8 configuration file should ignore line limit errors (E501).

### PR DESCRIPTION
**Pull request by SkullTech**
Fixes #9

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+language: python
+python:
+  - "3.6"
+# command to install dependencies
```
Comment (marco-c) [inline .travis.yml]: Nit: remove this and the other comment, as the yaml keys are self-explanatory.

File: setup.cfg [hunk 2]
```diff
@@ -1,3 +1,6 @@
 [flake8]
```
Comment (marco-c) [inline setup.cfg]: Oh, I forgot setup.cfg was already in the repo :)

File: setup.cfg [hunk 3]
```diff
@@ -1,3 +1,6 @@
...
-#ignore line limit error
-ignore = E501
\ No newline at end of file
+# no need to traverse these directories
+exclude = .git,__pycache__
```
Comment (marco-c) [inline setup.cfg]: Can you add `data`, `data_resized` and `tools` too to the exclude list?

File: setup.cfg (hunk 2 above)
Comment (SkullTech) [reply setup.cfg]: Yeah :)

File: .travis.yml (hunk 1 above)
Comment (SkullTech) [reply .travis.yml]: Yeah now that you say it, they are pretty redundant.

**Comment by marco-c** [comment]
Thanks!

**Pull request merged**